collab-entity/src/proto/collab.rs

```diff
@@ -1,32 +1,4 @@
 // This file is @generated by prost-build.
-/// Identifier of an active collab document sent over pubsub
-#[allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
why do you modify these?

Reply:
this is auto generated

Reply:
it seems like auto format. try not to change it. After you change it, i build, then they revert back